### PR TITLE
23

### PR DESCRIPTION
**Pull request by partouf**

### CODE REVIEW

File: GitBitterLib/Implementations/GitSharpCloner.cs [hunk 1]
```diff
@@ -123,27 +131,68 @@ public Task ResetAndUpdateExisting(string repository, string rootdir, string rep
...
+            else
+            {
+                var credentials = new SshUserKeyCredentials();
+                credentials.Username = "git";
+                credentials.Passphrase = string.Empty;
+                credentials.PrivateKey = privkey;
+                credentials.PublicKey = pubkey;
 
-            return (_url, _user, _cred) => credentials;
+                return (_url, _user, _cred) => credentials;
```
Comment (martinvd) [inline GitBitterLib/Implementations/GitSharpCloner.cs]: Both returns are same and can be placed below if-statement. Same for `credentials.Username = "git";`.

**Review by martinvd — CHANGES_REQUESTED** [review]

**Comment by martinvd** [comment]
"martinvd requested changes just now" lol :smile:

Anyway, you can consider making the change, otherwise just merge :+1:

File: GitBitterLib/Implementations/GitSharpCloner.cs (hunk 1 above)
Comment (partouf) [reply GitBitterLib/Implementations/GitSharpCloner.cs]: yes, need to figure out what the base class is of both credential objects though.. but, will do

**Comment by partouf** [comment]
Base class doesn't have all the properties, so it would be a mess of typecasting vs what it is now

**Pull request merged**